.drone.yml

```diff
@@ -0,0 +1,18 @@
+build:
+  image: gregway/gbm_immune_validation:latest
+  commands:
+    - bash run_pipeline.sh
+
+    - git config user.email "gregory.way@gmail.com"
+    - git config user.name "gwaygenomics"
+    - git config --global push.default simple
+    - git remote set-url origin https://github.com/greenelab/gbm_immune_validation
+
+    - git checkout master
+    - git add *
+    - git commit -a -m "Drone Build [skip ci] [ci skip]"
+    - git remote set-url origin https://gwaygenomics:$$GIT_PUBLISH_KEY@github.com/greenelab/gbm_immune_validation.git
```

Comment:
@brettbj does this `GIT_PUBLISH_KEY` work only for this repo, or does it work for all of @gwaygenomics repos. One consideration is that you probably want to create tokens with the minimal rights needed in case there's a leak.